README.txt

```diff
@@ -1,23 +1,30 @@
-This is JODReports version 2.4.0, released on 2011-01-28
+This is JODReports version 4.0.0 development by Barry Pearce, a fork of the 
```

Comment:
Same here: please remove the notion of a fork for this repo.

If you like (and considering the amount of work you invested here) feel free to add yourself as contributor in the Readme!